### DISCUSSION TITLE
Resync Configuration

### DISCUSSION THREAD

**Discussion by xCykrix** [repository-settings/app#543] [Q&A] [answered]
Hey,

I was curious if there is a viable way for this project to resync configurations if things are not in sync later on in time. I extend from my organizations `.github/settings.yml` and haven't been able to figure out how to sync settings properly in testing. Does settings only trigger on update of the settings.yml or is there another way for it to sync from the extended configuration?

The repositories I've been testing with are https://github.com/amethyst-studio/.github and https://github.com/amethyst-studio/resources

**Comment by travi** [comment, marked as answer]
currently, the primary trigger is an update to the `settings.yml` in the repository. there are a few other triggers that I hope to add in the future, but have not had the change to make progress there yet.

**Reply by xCykrix** [reply]
Well, that is unfortunate. Would you be open to looking for assistance to add a few of those triggers? I'd definitely be open to looking into assisting with that as I would love to have more options for synchronization from the .github repository to the entire organization's repositories. I'm not sure if there are any existing issues or a list of the ones you'd like to see as additional triggers.

From what I could see, the currently supported is only
- `push` with a change to `settings.yml`
- `repository.edited` that changed the default branch name?
